free-programming-books-pl.md

```diff
@@ -123,6 +125,7 @@
 ### MySQL
 
 * [Bazy Danych MYSQL](https://miroslawzelent.pl/kurs-mysql/)
+* [Kurs MYSQL](http://webmade.org/kursy-online/kurs-mysql.php)
```

Comment:
hint - if the title is "kurs-msql" maybe it's a "kurs"

Reply:
ok